vocabularies/Common.json

```diff
@@ -1542,6 +1561,17 @@
             "@Core.IsURL": true,
             "@Core.Description": "URL for uploading new media content to a Document Management Service",
             "@Core.LongDescription": "In contrast to the `@odata.mediaEditLink` this URL allows to upload new media content without directly changing a stream property.\nThe upload request typically uses HTTP POST with `Content-Type: multipart/form-data` following RFC 7578.\nThe upload request must contain one multipart representing the content of the file. The `name` parameter in the `Content-Disposition` header (as described in RFC 7578) is irrelevant, but the `filename` parameter is expected.\nIf the request succeeds the response will contain a JSON body of `Content-Type: application/json` with a JSON property `readLink`. The newly uploaded media resource can be linked to the stream property by changing the `@odata.mediaReadLink` to the value of this `readLink` in a subsequent PATCH request to the OData entity."
+        },
+        "PrimitivePropertyPath": {
+            "$Kind": "Term",
+            "$Type": "Core.Tag",
+            "$DefaultValue": true,
+            "$AppliesTo": [
+                "Term",
+                "Property"
+            ],
+            "@Common.Experimental": true,
+            "@Core.Description": "A term or term property with this tag whose type is (a collection of) `Edm.PropertyPath` MUST resolve to a primitive structural property of an entity type"
```

Comment:
Oh no, not another meta annotation!

Reply:
@BerSie, you asked for this restriction to "primitive property paths", would you agree if we specify it only in the descriptions rather than through a new "meta-annotation"?

Reply:
Yes, that's for me sufficient.

Reply:
How many other use cases do we have for this new meta-annotation, i.e. on which other terms/properties of type `Edm.PropertyPath` would we need to add this?

Would this only affect SAP vocabularies, or do we see the need for this also in OASIS vocabularies?

Reply:
Ok, in that case I think it is helpful to add this meta-annotation.